closeio/contrib/testing_stub.py

```diff
@@ -36,6 +41,54 @@ def _data(self, attr, default=None):
 
         return storage[attr]
 
+    def _create_task_log(self, task):
+        return {
+            'id': 'ev_{}'.format(uuid.uuid4().hex),
+            'object_type': 'task.lead',
+            'object_id': task['id'],
+            'lead_id': task['lead_id'],
+            'user_id': task['assigned_to'],
+            'date_updated': datetime.utcnow().isoformat(),
+        }
+
+    def _create_task_log_created(self, task):
+        logs = self._data('event_logs', [])
+        log = self._create_task_log(task)
+        log.update({
+            'action': 'created',
+            'data': task,
+            'previous_data': {},
+        })
+        logs.insert(0, log)
+
+    def _create_task_log_deleted(self, task):
+        logs = self._data('event_logs', [])
+        log = self._create_task_log(task)
+        log.update({
+            'action': 'deleted',
+            'data': {},
+            'previous_data': task,
+        })
+        logs.insert(0, log)
+
+    @contextmanager
+    def record_logs(self):
```

Comment:
Sorry for coming back to this topic again but I was thinking about the bug you caught yesterday. Looking at the usage example of `record_logs`, I think that would be nice adding a test for it too. Something like:

```
    def test_record_logs(self):
        task = stub.create_task(lead_id='x', assigned_to='y', text='z')
        stub.delete_task(task_id=task['id'])
        
        logs = client.get_event_logs(object_type='task.lead')
        logs = list(logs)
        assert len(logs) == 0
        
        with stub.record_logs():
            task = stub.create_task(lead_id='x', assigned_to='y', text='z')
            stub.delete_task(task_id=task['id'])
        
        logs = client.get_event_logs(object_type='task.lead')
        logs = list(logs)
        assert len(logs) == 2
```

What do you think?